draft-ietf-rtcweb-jsep.xml

```diff
@@ -1713,13 +1796,38 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             </list>
           </t>
 
-          <t>If the initial offer was applied using setLocalDescription, but
-          an answer from the remote side has not yet been applied, meaning the
-          PeerConnection is still in the "local-offer" state, an offer is
-          generated by following the steps in the "stable" state above, along
+          <t>If the initial offer was applied using setLocalDescription,
```

Comment:
Did you mean to make this change? This is a significant change to the have-local-offer processing, which I don't think is correct.